### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by maxvolokhov**

### CODE REVIEW

File: fifth_task.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+names = ["John Dow", "John Smith", "Marta Dow", "Marta Jones", "John Smith"]
+unique_names = list(dict.fromkeys(names))
+print(unique_names)
```
Comment (bohdandovbsyh) [inline fifth_task.py]: image
You do not have empty line at the end of file

File: first_task.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+eleks = ['John', 'Viktor', 'Olha', 'Abraham']
+toshiba = ['John', 'Peter', 'Aleksandr', 'Julia']
+eleks.append(toshiba)
```
Comment (bohdandovbsyh) [inline first_task.py]: extend is better. the append will add eleks list as the last element to Toshiba.
also after your manipulations eleks still has employees. please fix it

File: fourth_task.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+vip_guests = ('Anna', 'John', 'Gary')
+common_guests = {'Jack', 'Peter', 'Olha'}
+print(f'Vip guests: {vip_guests}')
+print(f'Common guests: {common_guests}')
```
Comment (bohdandovbsyh) [inline fourth_task.py]: add empty line here

File: fourth_task.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+vip_guests = ('Anna', 'John', 'Gary')
+common_guests = {'Jack', 'Peter', 'Olha'}
```
Comment (bohdandovbsyh) [inline fourth_task.py]: where is empty places?

File: third_task.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+omnivores = {'John', 'Peter', 'Mary', 'Stew'}
+vegetarians = {'Anna', 'Peter', 'Alina', 'Mary'}
+guests_who_can_eat_vegetables = omnivores.intersection(vegetarians)
```
Comment (bohdandovbsyh) [inline third_task.py]: but omnivores can eat vegetables as well. And all vegetarians can eat vegetables

File: second_task.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+casino_blacklist = ['John Dow', 'Aleks Smith', 'Penelopa Cruz']
+poker_blacklist = ['John Dow', 'Daniel Welbeck', 'Jogan Girera']
+alcohol_blacklist = ['John Dow', 'Jenifer Black', 'Axel Sniffer']
+new_list = set(casino_blacklist).intersection(poker_blacklist).intersection(alcohol_blacklist)
```
Comment (bohdandovbsyh) [inline second_task.py]: no empty line

File: fifth_task.py (hunk 1 above)
Comment (maxvolokhov) [reply fifth_task.py]: All tasks have been fixed and new commits have been added.
Hope it visible from your good side.

File: fourth_task.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+vip_room = ('Anna', 'John', 'Gary')
+common_room = ['Jack', 'Peter', 'Olha']
```
Comment (bohdandovbsyh) [inline fourth_task.py:2]: I still do not see empty places in this room

File: fourth_task.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
...
+vip_seats = tuple()
+empty_seats = None
```
Comment (bohdandovbsyh) [inline fourth_task.py:4]: what is it line 3 and 4 for?

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**